assets/freedesktop/org.zealdocs.Zeal.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.zealdocs.Zeal.desktop</id>
+  <name>Zeal</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <summary>Documentation browser</summary>
+  <description>
+    <p>Zeal is a simple offline documentation browser inspired by Dash.</p>
+  </description>
+  <url type="homepage">https://zealdocs.org/</url>
+  <url type="bugtracker">https://github.com/zealdocs/zeal/issues</url>
+  <url type="help">https://zealdocs.org/usage.html</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://i.imgur.com/qBkZduS.png</image>
+    </screenshot>
+  </screenshots>
+  <provides>
+    <id>zeal.desktop</id>
```

Comment:
This file would be absent, wouldn't it?

Reply:
In case of distributions shipping older release of Zeal with custom appdata, it will cause Software/Discover to merge both and prefer the latest one.